relay-dynamic-config/src/global.rs

```diff
@@ -58,11 +66,32 @@ pub struct Options {
     #[serde(rename = "profiling.profile_metrics.unsampled_profiles.enabled")]
     pub unsampled_profiles_enabled: bool,
 
+    /// Kill switch for controlling the cardinality limiter.
+    #[serde(rename = "relay.cardinality-limiter.mode")]
+    pub cardinality_limiter_mode: CardinalityLimiterMode,
+
     /// All other unknown options.
     #[serde(flatten)]
     other: HashMap<String, Value>,
 }
 
+/// Kill switch for controlling the cardinality limiter.
+#[derive(Default, Clone, Copy, Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub enum CardinalityLimiterMode {
+    /// Cardinality limiter is enabled.
+    #[default]
+    // De-serialize from the empty string, because the option was added to
+    // Sentry incorrectly which makes Sentry send the empty string as a default.
+    #[serde(alias = "")]
+    Enabled,
+    /// Cardinality limiter is enabled but cardinality limits are not enforced.
+    Passive,
+    /// Cardinality limiter is disabled.
+    Disabled,
```

Comment:
Is there a chance this enum will be extended in the future? If so, we should add an `Unknown(String)` option, such that external Relays can still parse the config.